tools/server-overlay.yml

```diff
@@ -0,0 +1,15 @@
+####
+#### WARNING: WIP
+#### This playbook is for ??.??.
+####  * Add remote logging to an already established server on S3.
+####
+---
+- hosts: all
+  remote_user: ubuntu
+  become: yes
+  roles:
+    - role: logrotate-to-s3
+      var_s3_path: "obo/purl"
+      var_name_prefix: ""
+      var_s3_push_credentials: "/home/sjcarbon/local/share/secrets/bbop/aws/s3/.s3cfg.obo-service-log-push"
```

Comment:
This would need to be changed to the s3cfg file on your local system.

Reply:
Tag onto #63

Reply:
See: https://s3tools.org/kb/item14.htm for an example.
As well, I could send some examples, if useful.